package.json

```diff
@@ -101,21 +109,13 @@
     "example"
   ],
   "packageManager": "yarn@3.6.1",
-  "engines": {
-    "node": ">= 18.0.0"
-  },
   "jest": {
     "preset": "react-native",
     "modulePathIgnorePatterns": [
       "<rootDir>/example/node_modules",
       "<rootDir>/lib/"
     ]
   },
-  "commitlint": {
```

Comment:
we weren't using this, and it enforces a rather annoying commit message format that I don't expect our devs to follow, for the most part because we typically squash merge and format commit messages at that time.